SpectralCubeReprojectMaskExample_v3.ipynb

```diff
@@ -0,0 +1,1546 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "4.3.dev1788+ga3263b6\n",
+      "0.8.dev31+g9f8ec7b\n"
+     ]
+    }
+   ],
+   "source": [
+    "import astropy\n",
+    "print(astropy.__version__)\n",
+    "import reproject\n",
+    "print(reproject.__version__)\n",
+    "# Need development version of astropy\n",
+    "# 4.3.dev1788+ga3263b6 works for me"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: PossiblySlowWarning: This function (<function BaseSpectralCube.reproject at 0x2ad7b51e38c8>) requires loading the entire cube into memory and may therefore be slow. [spectral_cube.utils]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "SpectralCube with shape=(75, 250, 250) and unit=K:\n",
+       " n_x:    250  type_x: RA---SIN  unit_x: deg    range:   266.534072 deg:  266.554577 deg\n",
+       " n_y:    250  type_y: DEC--SIN  unit_y: deg    range:   -28.713958 deg:  -28.695975 deg\n",
+       " n_s:     75  type_s: VRAD      unit_s: km / s  range:      -43.509 km / s:     104.685 km / s"
+      ]
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cube2vel_reproj = cube2vel_spatialspectralsmooth.reproject(cube1vel.header)\n",
+    "cube2vel_reproj "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These two cubes are now on an identical grid, and can be directly compared:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(SpectralCube with shape=(75, 250, 250) and unit=K:\n",
+       "  n_x:    250  type_x: RA---SIN  unit_x: deg    range:   266.534072 deg:  266.554577 deg\n",
+       "  n_y:    250  type_y: DEC--SIN  unit_y: deg    range:   -28.713958 deg:  -28.695975 deg\n",
+       "  n_s:     75  type_s: VRAD      unit_s: km / s  range:      -43.509 km / s:     104.685 km / s,\n",
+       " SpectralCube with shape=(75, 250, 250) and unit=Jy / beam:\n",
+       "  n_x:    250  type_x: RA---SIN  unit_x: deg    range:   266.534072 deg:  266.554577 deg\n",
+       "  n_y:    250  type_y: DEC--SIN  unit_y: deg    range:   -28.713958 deg:  -28.695975 deg\n",
+       "  n_s:     75  type_s: VRAD      unit_s: km / s  range:      -43.509 km / s:     104.685 km / s)"
+      ]
+     },
+     "execution_count": 26,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cube2vel_reproj, cube1vel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These spectra can now be overplotted as they are in the same unit with the same beam."
```

Comment:
Link with those peak maps from above to give an explanation for how you chose this slice of the data to show.